.ci/scripts/test_eval_llama_mmlu.sh

```diff
@@ -35,6 +35,7 @@ run_and_verify() {
         exit 1
     fi
     $PYTHON_EXECUTABLE -m examples.models.llama.eval_llama \
+	--model llama2 \
```

Comment:
this seems weird because we are actually running the stories model here but we use llama2 as the name.